app/components/MessageComposer/index.jsx

```diff
@@ -29,7 +29,7 @@ export default class MessageComposer extends React.Component {
     const text = this.state.text.trim();
     if (text) {
       this.props.newMessage({
-        id: 1,
+        // id: 1,
```

Comment:
Do not commit commented out code, especially without comments. In this case just remove it, will add it later if needed.